manifests/vanilla/deploy-csi-snapshot-components.sh

```diff
@@ -59,7 +59,7 @@ else
         exit 1
 fi
 
-qualified_version="v6.2.1"
+qualified_version="v6.2.2"
```

Comment:
@deepakkinni any concerns about this update?  6.2.2 contains CVE fixes.

Reply:
Not an issue. We need to run the pre-checkins to be sure though.